js/setup.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const NAMES = [
+  `Иван`,
+  `Хуан Себастьян`,
+  `Мария`,
+  `Кристоф`,
+  `Виктор`,
+  `Юлия`,
+  `Люпита`,
+  `Вашингтон`
+];
+const SURNAMES = [
+  `да Марья`,
+  `Верон`,
+  `Мирабелла`,
+  `Вальц`,
+  `Онопко`,
+  `Топольницкая`,
+  `Нионго`,
+  `Ирвинг`
+];
+const COAT_COLOR = [
+  `rgb(101, 137, 164)`,
+  `rgb(241, 43, 107)`,
+  `rgb(146, 100, 161)`,
+  `rgb(56, 159, 117)`,
+  `rgb(215, 210, 55)`,
+  `rgb(0, 0, 0)`,
+];
+const EYES_COLOR = [
+  `black`,
+  `red`,
+  `blue`,
+  `yellow`,
+  `green`
+];
+const NUMBER_PLAYERS = 4;
+
+const setup = document.querySelector(`.setup`);
+const similarWizards = setup.querySelector(`.setup-similar`);
+const similarWizardsList = setup.querySelector(`.setup-similar-list`);
+const similarWizardsTemplate = document.querySelector(`#similar-wizard-template`).content.querySelector(`.setup-similar-item`);
+
+const showElement = function (element) {
+  element.classList.remove(`hidden`);
+};
+
+const getRandomData = function (name) {
+  return name[Math.floor(Math.random() * name.length)];
+};
+
+const createArray = function (numbers) {
```

Comment:
```suggestion
const createDataArray = function (numbers) {
```
Лучше явно заявлять что конкретно возвращает функция, так в будущем не придется к ней возвращаться и разбираться